src/jit/rationalize.cpp

```diff
@@ -157,6 +157,13 @@ void Rationalizer::RewriteNodeAsCall(GenTree**             use,
     // Create the call node
     GenTreeCall* call = comp->gtNewCallNode(CT_USER_FUNC, callHnd, tree->gtType, args);
     call              = comp->fgMorphArgs(call);
+    // Determine if this call has changed any codegen requirements.
+    comp->fgCheckArgCnt();
+    if ((call->gtFlags & GTF_EXCEPT) != 0)
+    {
+        comp->codeGen->setFramePointerRequiredEH(true);
+    }
```

Comment:
Actually I don't understand the reason that you check for GTF_EXCEPT flags on a call node here.
A comment explaining this is needed. (Typically every user  function call should have this flag set)